src/components/SearchResultDrawer/index.tsx

```diff
@@ -141,7 +195,35 @@ export const SearchResultDrawer: React.FC<SearchResultDrawerProps> = ({
               dataIndex: 'attributeValue',
               key: 'attributeValue',
               width: '50%',
-              ellipsis: true
+              ellipsis: true,
+              render: (value: any) => {
+                let normalized = value;
+
+                if (
+                  Array.isArray(value) &&
+                  value.length === 1 &&
+                  typeof value[0] === 'string'
+                ) {
+                  normalized = value[0];
+                }
+
+                if (
+                  typeof normalized === 'string' &&
+                  isUrl(normalized) &&
+                  searchFeatureConfig
+                ) {
+                  return (
+                    <a
+                      href={normalized}
+                      target="_blank"
+                    >
+                      {t('FeaturePropertyGrid.linkText')}
+                    </a>
+                  );
+                }
+
+                return normalized ?? '';
+              }
```

Comment:
This might me be better in its own component to enhance readability.

Reply:
Right, I have created `AttributeValueCell` which will be used here instead.